.travis.yml

```diff
@@ -8,22 +8,32 @@ script:
   - npm run build
 jobs:
   include:
-  - stage: "Fundamental-react: Test & Lint"
+  - stage: Lint and Test
     script: npm run lint && npm run test:coveralls && npm run size
-  - stage: Deploy
-    if: (branch = master OR branch = tmp_branch_for_automated_release_do_not_use) AND type = push
+  - stage: Pre-release
+    if: branch = master
```

Comment:
We should  add `AND type = push` here so that it doesn't do a pre-release when the PR travis run builds.

Reply:
Nevermind! Travis is smarter than I thought it was 🤓